src/main/resources/configCivcraft.yml

```diff
@@ -9487,7 +9472,7 @@ recipes:
     input:
       emeralds:
         material: EMERALD
-        amount: 5
+        amount: 10
```

Comment:
I think Max agrees he just felt 5e was too dirt cheap, but as he said it's still really cheap at 10e.